nuget.config

```diff
@@ -5,7 +5,6 @@
   -->
     <packageSources>
         <clear />
-        <add key="LocalTest" value="c:\projects\sitecore.habitathome.commerce\lib\LocalRepository" />
```

Comment:
@jeanfrancoislarente The local source 'c:\projects\sitecore.habitathome.commerce\lib\LocalRepository' doesn't exist.